cli/tests/it/cmd.rs

```diff
@@ -426,8 +433,24 @@ warning[5667]: Warning: Unused function parameter. Remove or comment out the var
     ));
 });
 
-// tests that direct import paths are handled correctly
-forgetest!(canhandle_direct_imports_into_src, |prj: TestProject, mut cmd: TestCommand| {
+// Tests that direct import paths are handled correctly
+//
+// NOTE(onbjerg): Disabled for Windows -- for some reason solc fails with a bogus error message
```

Comment:
I really have NO IDEA at all what is going on here. "Invalid implicit conversion from struct Bar memory to struct Bar memory"????